ejercicio1vulky/src/main/java/com/mycompany/ejercicio1vuky/ModeloPersona.java

```diff
@@ -0,0 +1,48 @@
+
+package com.mycompany.ejercicio1vuky;
+
+
+public class ModeloPersona{
+    
+ private String colorP;
```

Comment:
What is colorP?

Please rename to a more clear naming variable

Reply:
ready.